src/index.js

```diff
@@ -64,6 +66,15 @@ async function bundleMDX(
           if (fullModulePath in absoluteFiles) return {path: fullModulePath}
         }
 
+        if(cwd){
+          const cwdModulePath = modulePath.replace(/^(.*__mdx_bundler_fake_dir__)/, cwd)
+
+          for (const ext of ['', '.js', '.ts', '.jsx', '.tsx', '.json', '.mdx']) {
```

Comment:
starting with `''` in the array adds support for someone directly linking to an asset like `./index.css` or `./image.png` and returns staright away before trying all the others.

Reply:
I'm not sure I understand this. Wouldn't `ext` be `.css` or `.png` in that case?

Reply:
I wanted to add a blanket just test the unmodified file name first so that `import './index.css` would work.

I'm bascially doing the same as the check to see if there is an entry in the files object but with `fs.existsSync` so this array is doing the same thing of testing module extensions to find the file, adding `''` is just a quick way of testing the exact name as a file.